### ISSUE TITLE
Dedicated vs. Low-priority servers

### ISSUE THREAD

**Issue by rzhao01** [microsoft/AzureML-BERT#36]
Hi, I just have a question. Does this code handle being pre-empted on Azure ML low-priority VMs? I've read that BERT is particularly sensitive and requires the model, optimizer state, and dataset shuffling to all be saved and restored if pre-empted during pre-training.
